src/test/java/bio/terra/common/TestUtils.java

```diff
@@ -326,4 +329,21 @@ public static String readControlFile(List<BulkLoadFileModel> bulkLoadFileModelLi
     }
     return sb.toString();
   }
+
+  /**
+   * Asserts that a command fails with the type passed in with a message that contains the expected
+   * message
+   *
+   * @param expectedThrowable The class that should be thrown. Note: this is the top level exception
+   *     and not the cause exception
+   * @param expectedMessage The partial message that should be present in the exception
+   * @param runnable The code to test
+   */
+  public static void assertError(
```

Comment:
Oh fun, I can think of a few older tests I've written that would use this.

Reply:
right?  Our assertion library doesn't handle exception matching super well but this should at least hide that some